os/arch/arm/src/armv8-m/up_assert.c

```diff
@@ -491,6 +491,7 @@ void up_assert(const uint8_t *filename, int lineno)
 	} else {
 		assert_pc = (uint32_t)g_assertpc;
 	}
+	lldbg("g_assertpc: %08x\n", g_assertpc);
```

Comment:
1) I will share these results.
2) Thank you for pointing it out, I think it is better to move it after the CONFIG_APP_BINARY_SEPARATION section.
3) So I kept it similar to below so that we can extract using the same symbol name:
        lldbg("sp:         %08x\n", sp);
        lldbg("stack base: %08x\n", ustackbase);
        lldbg("stack size: %08x\n", ustacksize);

It gives something as below:
up_assert: g_assertpc : 02100df9
up_dumpstate: sp:         02103d90
up_dumpstate: stack base: 02103e70
up_dumpstate: stack size: 00001fe4

We can rename it though. Please let me know what fits better.

Reply:
sp, stack base, stack size are commonly used. but g_assertpc is not. asserted pc or something else is better I think.